src/main/frontend/components/StaticStarRating.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import Star from './Star.js'
+
+const StaticStarRating = (props)=>  {
+ 
+  return (
+    <div>
+      {Array.from({ length: props.starRating }, (i) => (
+        <Star
+          key={`star_${i + 1} `}
+          marked={true}
+        />
+      ))}
+      {Array.from({ length: 5 - props.starRating }, (i) => (
+        <Star
+          key={`star_${i + 1} `}
+          marked={false}
+        />
+      ))}

```

Comment:
Again I would move the array construction above the return statement and save it to a variable.